src/System.Memory/src/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard1.0;
+      netstandard1.0-Windows_NT;
```

Comment:
It was excluded in src.builds, I just moved that exclusion here.  @mellinoe  may have more context.